### PR TITLE
Tickets and orders

### PR DESCRIPTION
**Pull request by Ananiev-Vitalii**

### CODE REVIEW

File: services/user.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+def create_user(
+        username: str, password: str, email: str | None = None,
+        first_name: str | None = None, last_name: str | None = None
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email if email is not None else "",
+        first_name=first_name if first_name is not None else "",
+        last_name=last_name if last_name is not None else ""
```
Comment (vsmutok) [inline services/user.py:16]:
```suggestion
        email=email or "",
        first_name=first_name or "",
        last_name=last_name or ""
```
better use or in this case

File: services/user.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    )
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```
Comment (vsmutok) [inline services/user.py:22]: it is better to use get_user_model() method instead of User
fix this everywhere

File: services/user.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+@transaction.atomic
+def update_user(
+        user_id: int,
+        username: str | None = None,
+        password: str | None = None,
+        email: str | None = None,
+        first_name: str | None = None,
+        last_name: str | None = None
+) -> None:
+    user = User.objects.get(id=user_id)
```
Comment (vsmutok) [inline services/user.py:34]: u can use get user method here

**Review by vsmutok — CHANGES_REQUESTED** [review]

File: services/user.py (hunk 2 above)
Comment (Ananiev-Vitalii) [reply services/user.py:22]: @vsmutok User = get_user_model() But that's what I use?

**Pull request closed without merging**